src/components/RouteOption/DockedEbikeRouteOptionDetails.jsx

```diff
@@ -33,16 +33,28 @@ function DockedEbikeRouteOptionDetails({ routeOption }) {
     walkingTimeFromStartingPoint,
     cyclingTimeStationToStation,
     walkingTimeToDestination,
+    usualAvailabilityAtBikePickupStation,
+    usualAvailabilityAtBikeDropOffStation,
   } = routeOption.details;
 
+  const pickupStationCapacity =
+    (usualAvailabilityAtBikePickupStation?.standardBikes || 0) +
+    (usualAvailabilityAtBikePickupStation?.electricBikes || 0) +
+    (usualAvailabilityAtBikePickupStation?.emptyDocks || 0);
+
+  const dropOffStationCapacity =
+    (usualAvailabilityAtBikeDropOffStation?.standardBikes || 0) +
+    (usualAvailabilityAtBikeDropOffStation?.electricBikes || 0) +
+    (usualAvailabilityAtBikeDropOffStation?.emptyDocks || 0);
```

Comment:
Thinking about it, though 🤔 Usual availability is indeed nullable, i.e., the backend may 'legally' return `null` if it has no corresponding historical availability data.

With the current implementation, we will show availability 0, which seems incorrect. Could we show something else to distinguish between availability 0 and missing availability? A question mark, maybe? Or a hyphen.